javascript/javascript1/week4/characterFrequency.js

```diff
@@ -0,0 +1,19 @@
+function getFrequency(string) 
```

Comment:
`string` is a special keyword, i don't recommend you name variables like this. `word` is a better choice for example.